docs/ci-cd-environment/nested-virtualization.md

```diff
@@ -0,0 +1,46 @@
+# Nested virtualization
+
+Linux based virtual machines support nested virtualization.
+
+To check that nested virtualization is supported , we suggest
```

Comment:
I would say "To check if nested..." not "To check that nested..."

Reply:
Changed as suggested